pallets/schemas/src/lib.rs

```diff
@@ -78,40 +79,89 @@ pub mod pallet {
 	#[pallet::generate_store(pub(super) trait Store)]
 	pub struct Pallet<T>(_);
 
+	#[pallet::storage]
+	#[pallet::getter(fn get_schema_max_bytes)]
+	pub(super) type GovernanceSchemaMaxBytes<T: Config> = StorageValue<_, u32, ValueQuery>;
```

Comment:
I think eventually we might have a separate pallet for governance but for now this seems fine